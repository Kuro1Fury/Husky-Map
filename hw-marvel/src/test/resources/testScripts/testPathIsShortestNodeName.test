# Test that the path is the shortest based on node name

CreateGraph A

AddNode A n1
AddNode A n2
AddNode A n3
AddNode A n4
AddEdge A n1 n2 Edge
AddEdge A n2 n4 Edge
AddEdge A n3 n4 Edge
AddEdge A n1 n3 Edge

FindPath A n1 n4
