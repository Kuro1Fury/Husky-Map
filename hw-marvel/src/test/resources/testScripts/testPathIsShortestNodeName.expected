# Test that the path is the shortest based on node name

created graph A

added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A
added edge Edge from n1 to n2 in A
added edge Edge from n2 to n4 in A
added edge Edge from n3 to n4 in A
added edge Edge from n1 to n3 in A

path from n1 to n4:
n1 to n2 via Edge
n2 to n4 via Edge